src/rpc/rawtransaction.cpp

```diff
@@ -1038,13 +1039,15 @@ static const CRPCCommand commands[] =
 { //  category              name                      actor (function)         argNames
   //  --------------------- ------------------------  -----------------------  ----------
     { "rawtransactions",    "getrawtransaction",      &getrawtransaction,      {"txid","verbose","blockhash"} },
+    { "rawtransactions",    "searchrawtransactions",  &searchrawtransactions,  {"address","verbose","skip","count"}  },
     { "rawtransactions",    "createrawtransaction",   &createrawtransaction,   {"inputs","outputs","locktime","replaceable"} },
```

Comment:
searchrawtransactions best be grouped with getaddressbalance below rather than in the rawtransactions group